client/package.json

```diff
@@ -9,6 +9,7 @@
     "axios": "^0.25.0",
     "bootstrap": "^5.1.3",
     "react": "^17.0.2",
+    "react-bootstrap": "^2.1.1",
```

Comment:
I am using react-bootstrap v5 which is the latest version. I guess it might not cause any confliction, but if you don't mind I will update the version of it later. :)